tests/examples-smoke-tests/package.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "@keystonejs/examples-smoke-tests",
+  "description": "A set of tests for smoke testing our examples",
+  "private": true,
+  "version": "9.0.0",
```

```suggestion
  "version": "0.0.0",
```
Should we start at `0.0.0` and let the changeset bot do its thing? We probably want a changeset on this PR as well in that case.

---

Updated it to 0.0.0 but I'm not going to add a changeset because this package isn't published so it does not matter.